src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,15 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+(defn parse-sequence-number-within-year-to-number [external-id]
+  (str-to-int (get (str/split external-id #"/") 1)))
+
+;; put test here
+(deftest test-validate-external-id
+  (is (= 10 (parse-sequence-number-within-year-to-number "2020/10")))
+  (is (= 6 (parse-sequence-number-within-year-to-number "2020/6"))))
```

Comment:
I don't see a case for `nil`, for different years, and different formats like `THL/2492/4.00.00/2020`

Reply:
is there any place I can look at other possible formats?

Reply:
I searched some with Google and several are in this format.